hack/lint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e -x -u
+
+GO111MODULE=off go get -u github.com/kisielk/errcheck
```

Comment:
any reason why we are disabling go modules here? soon that will be a required mode afaik.

Reply:
It should be installed globally not as a module in the workspace.

Reply:
hmm, i guess this assumes that just by installing something with go it becomes available on the PATH. not sure if other folks have it set up that way, but in my case, above command does not make things show up on the path since my global $GOPATH/bin isnt part of PATH. if we are going to rely on GOPATH installed binary lets change line below to use something like `${GOPATH:-$(go env GOPATH)}/bin/errcheck`.

Reply:
Adding `GOPATH/bin` to your PATH is part of the go installation [instructions](https://golang.org/doc/install#install). Also [`go install`](https://golang.org/cmd/go/#hdr-Compile_and_install_packages_and_dependencies) rely on that. I'd prefer to follow it it's a no-brainer.

Reply:
But sure, we can do it :smile: